sway/commands/workspace.c

```diff
@@ -181,7 +181,12 @@ struct cmd_results *cmd_workspace(int argc, char **argv) {
 			++argv;
 		}
 
-
+		bool create = false;
+		if (argc > 1) {
+			create = strcasecmp(argv[1], "--create") == 0;
+		}
+		struct sway_seat *seat = input_manager_current_seat();
```

Comment:
Since GitHub won't let me add the comment to the actual lines, can you change all ocurrances of seat to be the same for consistency? seat is redefined in the body for `back_and_forth` (L211) and `config->handler_context.seat` is used directly in `seat_consider_warp_to_focus` on L227.

Since this is in a command handler, `config->handler_context.seat` should be set, but `input_manager_current_seat()` should return `config->handler_context.seat` when set. I don't really care which is used, but it should be consistent